### PR TITLE
Removes canvases on runtime

### PR DESCRIPTION
**Pull request by Mothblocks**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl:
del: Temporarily removes canvases until someone figures out how to fix them.
/:cl:

[why]: # #38942 I didn't make it a config option because if you're going to be fixing the canvases you'd be removing the code for the config option anyway.

### CODE REVIEW

File: code/game/objects/structures/artstuff.dm [hunk 1]
```diff
@@ -66,6 +65,14 @@ GLOBAL_LIST_INIT(globalBlankCanvases, new(AMT_OF_CANVASES))
 	icon_state = "23x23"
 	whichGlobalBackup = 4
 
+//HEY YOU
+//ARE YOU READING THE CODE FOR CANVASES?
+//ARE YOU AWARE THEY CRASH HALF THE SERVER WHEN SOMEONE DRAWS ON THEM...
+//...AND NOBODY CAN FIGURE OUT WHY?
+//THEN GO ON BRAVE TRAVELER
+//TRY TO FIX THEM AND REMOVE THIS CODE
+/obj/item/canvas/New()
```
[the remark this answers is not in the document]
Comment (Jared-Fogle) [reply code/game/objects/structures/artstuff.dm]: Only because I don't think it makes much of a difference. I'll comment it out, though.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Remove the comment block and add ..() before returning the initialize hint

File: code/game/objects/structures/artstuff.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (Cyberboss) [reply code/game/objects/structures/artstuff.dm]: we don't comment shit out kevinz, you know this by now

**Comment by Jared-Fogle** [comment]
@Cyberboss Which comment block? The one with the rest of the canvas code?

**Comment by Cyberboss** [comment]
Yes

**Comment by Jared-Fogle** [comment]
Done

**Review by Cyberboss — APPROVED** [review]

**Comment by SgtxRootbeer** [comment]
so, has this been worked on at all yet?

**Pull request merged**